cluster/gce/coreos/kube-manifests/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -71,3 +71,4 @@ spec:
         emptyDir: {}
       - name: grafana-persistent-storage
         emptyDir: {}
+
```

Comment:
Intentional; it matches the ./cluster/addon one to make the diff slightly shorter.